src/modules/AccordionGrommet.js

```diff
@@ -0,0 +1,63 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from 'grommet/components/Box';
+import Headline from 'grommet/components/Headline';
+import List from 'grommet/components/List';
+import Paragraph from 'grommet/components/Paragraph';
+
+const CLASS_ROOT = 'accordion-grommet';
```

Comment:
Just wondering, why did you name this component/class root `AccordionGrommet`?  It should just be `Accordion`, unless there's a need to differentiate it from an already-existing Accordion component.